clearmash/pipeline-spec.yaml

```diff
@@ -124,6 +125,15 @@ entities-sync:
     - run: dump.to_path
       parameters:
         out-path: ../data/clearmash/new-entities-sync
+    # dump geocode_location table
+    - run: dump.to_sql
```

Comment:
I think it will be better to have the sync processor update the geocode_entities table